submit-web/src/routes/_authenticated/registration/create-account.tsx

```diff
@@ -56,6 +59,9 @@ function CreateAccount() {
   const methods = useForm({
     resolver: yupResolver(createAccountSchema),
     mode: "onBlur",
+    defaultValues: {
+      phone: "(000) 000-0000",
```

Comment:
I think we should do something else here, why do we need to set the default value like that, can't we set a placeholder in the textfield?

Reply:
ill do that